.ci/jenkins_job_build.sh

```diff
@@ -194,6 +194,9 @@ if [ -n "${METRICS_CI}" ]; then
 	echo "Running the metrics tests:"
 	"${tests_repo_dir}/.ci/run_metrics_PR_ci.sh"
 elif [ -n "${VFIO_CI}" ]; then
+	echo "Installing initrd image:"
+	TEST_INITRD=yes "${ci_dir_name}/install_kata_image.sh"
```

Comment:
Since you passing the `TEST_INITRD=yes` to the installation script, perhaps you should export it to the environment because other scripts eventually called later can need that information. So I suggest this:

```
export TEST_INITRD=yes
"${ci_dir_name}/install_kata_image.sh"
```

Reply:
Thanks @wainersm, (is this the dismissed review? I can't tell)

I don't think other scripts use this variable right now, and we don't necessarily want all of them to have `TEST_INITRD` set to `yes`. However, I made these edits on @devimc's suggestion, so perhaps he can comment.

Reply:
> Thanks @wainersm, (is this the dismissed review? I can't tell)

Yes, I wanted to make this single comment but I pressed the wrong button :)
> 
> I don't think other scripts use this variable right now, and we don't necessarily want all of them to have `TEST_INITRD` set to `yes`. However, I made these edits on @devimc's suggestion, so perhaps he can comment.

Ok, I don't know those scripts in deep and this is the only thing that caught on my eyes.

Reply:
@cmaf  you're right, however I don't see any inconvenience if you apply the changes suggested by @wainersm

Reply:
@devimc @wainersm I'll leave this as it is for now, thanks again for the feedback.